packages/starksheet-cairo/utils/deployment.py

```diff
@@ -269,7 +269,7 @@ def compile_contract(contract_name):
         raise ValueError(f"Cannot find src/**/{contract_name}.cairo in {os.getcwd()}")
     output = subprocess.run(
         [
-            "starknet-compile",
+            "starknet-compile-deprecated",
```

Comment:
looks like "starknet-compile" has been renamed to "starknet-compile-deprecated". I did not find online resources describing this, however it probably relates to the forecoming release of cairo 1, given the big changes in design it induces

Reply:
not sure about this as well, re-installing the venv from scratch, using `starknet-compile 0.10.0` I'm still able to run the command. This however should be updated to either starknet-py or something more regular though